frontend/ios/Podfile

```diff
@@ -42,5 +42,11 @@ end
 post_install do |installer|
   installer.pods_project.targets.each do |target|
     flutter_additional_ios_build_settings(target)
+        target.build_configurations.each do |config|
+            config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= [
+            '$(inherited)',
+            'PERMISSION_CAMERA=1',
+          ]
+        end
```

Comment:
Did you add this manually?

Reply:
yes this had to be added manually for ios